### ISSUE TITLE
Can't convert multi-line paragraphs to bullets

### ISSUE THREAD

**Issue by HerbCaudill** [mdickin/vscode-markdown-shortcuts#53]
When you have multi-line paragraphs like this

```markdown
Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Phasellus hendrerit.
Pellentesque aliquet nibh nec urna. In nisi neque, aliquet vel, dapibus id,
mattis vel, nisi.

Sed pretium, ligula sollicitudin laoreet viverra, tortor libero sodales leo,
eget blandit nunc tortor eu nibh. Nullam mollis. Ut justo.
```

and you select them and choose "Toggle bullet points", you get this: 

```markdown
- Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Phasellus hendrerit.
- Pellentesque aliquet nibh nec urna. In nisi neque, aliquet vel, dapibus id,
- mattis vel, nisi.

- Sed pretium, ligula sollicitudin laoreet viverra, tortor libero sodales leo,
- eget blandit nunc tortor eu nibh. Nullam mollis. Ut justo.
```

Instead you should get this: 

```markdown
- Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Phasellus hendrerit.
  Pellentesque aliquet nibh nec urna. In nisi neque, aliquet vel, dapibus id,
  mattis vel, nisi.

- Sed pretium, ligula sollicitudin laoreet viverra, tortor libero sodales leo,
  eget blandit nunc tortor eu nibh. Nullam mollis. Ut justo.
```

Currently the only way to get that is to add the bullets manually.

**Comment by mdickin** [comment]
Hi @HerbCaudill, I'm not sure how I would distinguish between your example and something like
```
# TODO
Clean the house. It's really dirty
Brush your teeth. Healthy gums are very important
Clean the catbox

Believe in yourself
Drink more water
```

I think this scenario would be more common than paragraphs with newlines in them. If you come up with a reliable way to do this, I'm more than willing to review a PR :)

**Comment by HerbCaudill** [comment]
Thanks, @mdickin . I guess my thinking was that you'd use [Markdown rules](https://daringfireball.net/projects/markdown/syntax#p)  to detect line breaks: Two consecutive lines are treated as being in the same paragraph, unless you end the first one with two more more spaces. In most flavors of Markdown, your TODO list of five items would be rendered as two paragraphs: 

![image](https://user-images.githubusercontent.com/2136620/68855383-6aa62080-06de-11ea-85f8-2af13cb8c4a1.png)
https://gist.github.com/HerbCaudill/8dc8b425a2f09e72b046356c0bfbbf29

I think you're probably right, though, that your scenario is just as common. 

What if in these situations the command cycled through three states instead of just toggling bullets/no bullets? 
1. One bullet per line
2. One bullet per paragraph
3. No bullets

**Comment by mdickin** [comment]
@HerbCaudill, you're right, we could detect it by the two spaces at the end of the paragraph. I wasn't aware of this syntax until a few days ago (#54 ). I think that would make this a lot easier.
